### PR TITLE
feat: Changed logger from winston to tv2 mediatech logger.

### PR DESCRIPTION
**Pull request by leerberg**
Hi Kasper and Balte, we have decided to improve and simplify the logging in Sisyfos. This means that we have chosen to move away from https://www.npmjs.com/package/winston to a tv2media specific which dosen't include any other dependencies https://www.npmjs.com/package/@tv2media/logger.  We have removed all console logs and we are new using 5 log formats, error, warn, info, debug, trace. Futher details about the tv2media logging framework can be read in the project README. 

A big reason for this is to be able to log JSON and hereby make the logs more searchable in log management software e.g. ElasticSearch, Kibana).

### CODE REVIEW

**Review by baltedewit — APPROVED** [review]
The change looks good but I can't help but notice that the repository for the logger is not public as of now, it's not a big issue but it feels strange? Will it become public in the future?

**Comment by olzzon** [comment]
Would it be possible to wait until next release. I'm currently doing long time testing on the current Dev repo. And we should release it soon. Then we could merge afterward.

**Comment by KvelaGorrrrnio** [comment]
Before merging, we should bump the logger package version to the one that will be generated from this PR: https://github.com/tv2/mediatech-logger/pull/15

**Pull request merged**
